mithril-common/src/signable_builder/cardano_transactions.rs

```diff
@@ -183,17 +187,20 @@ mod tests {
 
     #[tokio::test]
     async fn test_compute_signable() {
-        let beacon = Beacon::default();
+        // Arrange
```

Comment:
Do we need all these comments? Maybe a line break is enough.

Reply:
We add them because we found that it's not so easy to identify each part of the test.